http_service/tests/test_bug_classification.py

```diff
@@ -159,6 +159,41 @@ def do_request():
     }
 
 
+def test_for_missing_bugs(client, responses):
+    existed_bugs_ids = [1602463, 1619699]
+    missing_bugs_ids = [1598744, 1615281, 1566486]
+    all_bug_ids = [*existed_bugs_ids, *missing_bugs_ids]
+
+    change_time = str(time.time())
+
+    responses.add(
+        responses.GET,
+        "https://bugzilla.mozilla.org/rest/bug?id=1566486,1598744,1602463,1615281,1619699&include_fields=id&include_fields=last_change_time",
+        status=200,
+        json={
+            "bugs": [
+                {"id": bug_id, "last_change_time": change_time}
+                for bug_id in existed_bugs_ids
+            ],
+        },
+    )
+
+    rv = client.post(
+        "/component/predict/batch",
+        data=json.dumps({"bugs": all_bug_ids}),
+        headers={API_TOKEN: "test"},
+    )
+    assert rv.status_code == 202
+    bugs = retrieve_compressed_reponse(rv)["bugs"]
+
+    assert list(bugs.keys()) == [
+        str(bug_id) for bug_id in all_bug_ids
+    ], "All the queried bug IDs must be returned"
```

Comment:
Instead of converting `bugs.keys()` to a list, we could compare it to a set. Also, since we need bug ids to be a list of strings, why not have them like that since the beginning?

Reply:
Well noted